resources/services/lightsail/databases.go

```diff
@@ -0,0 +1,507 @@
+			options.Region = c.Region
+		})
+		if err != nil {
+			return diag.WrapError(err)
+		}
+		res <- response.RelationalDatabases
+		if aws.ToString(response.NextPageToken) == "" {
+			break
+		}
+		input.PageToken = response.NextPageToken
+	}
+	return nil
+}
+func resolveDatabasesTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	r := resource.Item.(types.RelationalDatabase)
+	tags := make(map[string]string)
+	client.TagsIntoMap(r.Tags, tags)
+	return diag.WrapError(resource.Set(c.Name, tags))
+}
+func fetchLightsailDatabasePendingMaintenanceActions(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	r := parent.Item.(types.RelationalDatabase)
+	res <- r.PendingMaintenanceActions
+	return nil
+}
+func fetchLightsailDatabaseParameters(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	r := parent.Item.(types.RelationalDatabase)
+	input := lightsail.GetRelationalDatabaseParametersInput{
+		RelationalDatabaseName: r.Name,
+	}
+	c := meta.(*client.Client)
+	svc := c.Services().Lightsail
+	for {
+		response, err := svc.GetRelationalDatabaseParameters(ctx, &input, func(options *lightsail.Options) {
+			options.Region = c.Region
+		})
+		if err != nil {
+			return diag.WrapError(err)
+		}
+		res <- response.Parameters
+		if aws.ToString(response.NextPageToken) == "" {
+			break
+		}
+		input.PageToken = response.NextPageToken
+	}
+	return nil
+}
+func fetchLightsailDatabaseEvents(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	r := parent.Item.(types.RelationalDatabase)
+	input := lightsail.GetRelationalDatabaseEventsInput{
+		RelationalDatabaseName: r.Name,
+		DurationInMinutes:      aws.Int32(20160), //two weeks
+	}
+	c := meta.(*client.Client)
+	svc := c.Services().Lightsail
+	for {
+		response, err := svc.GetRelationalDatabaseEvents(ctx, &input, func(options *lightsail.Options) {
+			options.Region = c.Region
+		})
+		if err != nil {
+			return diag.WrapError(err)
+		}
+		res <- response.RelationalDatabaseEvents
+		if aws.ToString(response.NextPageToken) == "" {
+			break
+		}
+		input.PageToken = response.NextPageToken
+	}
+	return nil
+}
+func fetchLightsailDatabaseLogEvents(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	r := parent.Item.(types.RelationalDatabase)
+	input := lightsail.GetRelationalDatabaseLogStreamsInput{
+		RelationalDatabaseName: r.Name,
+	}
+	c := meta.(*client.Client)
+	svc := c.Services().Lightsail
+	streams, err := svc.GetRelationalDatabaseLogStreams(ctx, &input, func(options *lightsail.Options) {
+		options.Region = c.Region
+	})
+	if err != nil {
+		return diag.WrapError(err)
+	}
+	endTime := time.Now()
+	startTime := endTime.Add(-time.Hour * 24 * 14) //two weeks
+	var sem = semaphore.NewWeighted(int64(MAX_GOROUTINES))
```

Comment:
nit: new errgroup has SetLimit method which frees us from using semaphore and simplifies the code.